packages/scandipwa/src/component/MyAccountMyWishlist/MyAccountMyWishlist.container.js

```diff
@@ -107,36 +107,38 @@ export class MyAccountMyWishlistContainer extends PureComponent {
         };
     }
 
-    addAllToCart() {
+    async addAllToCart() {
         const { moveWishlistToCart } = this.props;
 
         if (!isSignedIn()) {
-            return null;
+            return Promise.reject();
         }
 
         this.setState({ isLoading: true });
 
-        return moveWishlistToCart().then(
-            /** @namespace Component/MyAccountMyWishlist/Container/MyAccountMyWishlistContainer/addAllToCart/moveWishlistToCart/then */
-            () => this.showNotificationAndRemoveLoading('Available items moved to cart'),
-            /** @namespace Component/MyAccountMyWishlist/Container/MyAccountMyWishlistContainer/addAllToCart/moveWishlistToCart/then/catch */
-            (error) => this.showErrorAndRemoveLoading(getErrorMessage(error))
-        );
+        try {
+            return await moveWishlistToCart();
```

Comment:
1. Await is redudant here, since you return a Promise.
2. Since you return a value, catch will never run here.